### DISCUSSION TITLE
How do I capture mouse double-click events?

### DISCUSSION THREAD

**Discussion by pcevent** [slint-ui/slint#2772] [Q&A]
How do I capture mouse double-click events?

**Comment by tronical** [comment]
We have that information internally, but it's not exposed yet. I'll create a ticket.

**Reply by Jhonfunk** [reply]
any updates

**Reply by ogoffart** [reply]
The issue that was created is https://github.com/slint-ui/slint/issues/2774
But this hasn't been implemented yet.

**Comment by pcevent** [comment]
Okay, thank you for your reply.
